document-classification/content.md

```diff
@@ -1,3 +1,7 @@
+By classifying documents, we are aiming to assign one or more classes or categories to a document, making it easier to 
```

Comment:
By classifying documents, we are assigning one or more classes or categories to a document